django/contrib/admin/templates/registration/password_reset_email.html

```diff
@@ -1,5 +1,5 @@
 {% load i18n %}{% autoescape off %}
-{% blocktrans %}You're receiving this email because you requested a password reset for your user account at {{ site_name }}.{% endblocktrans %}
+{% blocktrans force_escape %}You're receiving this email because you requested a password reset for your user account at {{ site_name }}.{% endblocktrans %}
```

Comment:
@claudep @timgraham I re-checked code and think that since autoescape was disabled in the template, we should not enabled it for the translated string. Right?